tests/test_plugins.py

```diff
@@ -134,3 +134,9 @@ def test_timestamped_geo_json(self):
         mape = folium.Map([47, 3], zoom_start=1)
         mape.add_children(plugins.TimestampedGeoJson(data))
         mape._repr_html_()
+
+    def test_heat_map(self):
+        data = (np.random.normal(size=(100,2))*np.array([[1,1]])+np.array([[48,5]])).tolist()
+        mapa = folium.Map([48., 5.], tiles='stamentoner', zoom_start=6)
+        mapa.add_children(plugins.HeatMap(data))
+        mapa._repr_html_()
```

Comment:
Correct.

We can do a bit better in doing the same thing as `test_folium.py` : compare the output of `.render()` with code generated from a validation template.
But this is a heavy procedure and I'm not sure this is needed for all plugins (am I lazy ?).